### PR TITLE
GetProlongationMatrix returns IdentityOperator when NRanks ==1 [pfespace-fix]

### PR DESCRIPTION
**Pull request by camierjs**
Add IsIdentityProlongation in operator.hpp

| PR | Author | Editor | Reviewers  | Assignment | Approval | Merge |
| --- | --- | --- | ---  | --- | --- | --- |
| https://github.com/mfem/mfem/pull/1239 | @camierjs  | @tzanio | @v-dobrev + @pazner | 01/16/20 | 01/16/20 | ⌛due 01/23/20 |

### CODE REVIEW

File: linalg/operator.hpp [hunk 1]
```diff
@@ -415,6 +415,11 @@ class IdentityOperator : public Operator
    virtual void MultTranspose(const Vector &x, Vector &y) const { y = x; }
 };
 
+/// Returns true if the P operator identity.
```
Comment (pazner) [inline linalg/operator.hpp]:
```suggestion
/// Returns true if the P operator is identity.
```
A comment is also warranted here saying that historically `NULL` had been used to signify the identity operator as an optimization.

**Comment by v-dobrev** [comment]
For the serial version, similar change will not be as simple probably.

**Comment by tzanio** [comment]
@artv3, can you help us debug if this resolves the MPI issues on 1 processor?

**Comment by artv3** [comment]
> @artv3, can you help us debug if this resolves the MPI issues on 1 processor?

Sure thing! Testing is in progress.

**Comment by artv3** [comment]
Works for us!

**Comment by tzanio** [comment]
Thanks Arturo!

**Pull request merged**